controllers/change_coordinators.go

```diff
@@ -58,17 +58,6 @@ func (c changeCoordinators) reconcile(ctx context.Context, r *FoundationDBCluste
 		}
 	}
 
-	if status.Cluster.ConnectionString != cluster.Status.ConnectionString {
```

Comment:
I don't follow this fully. Are you saying updating connection string here is not needed? This is because you are now updating the connection string in "getStatusFromClusterOrDummyStatus()" function but that function is not getting called here.

Reply:
Having the update logic in one place is better than spring it across multiple places. There was an also a bug where the connection string was set to en empty string if the status had an empty connection string but the `GetStatus` was not returning an error. Updating the connection string here brings not a huge value and was probably a left over from some previous refactoring.